packages/sui-ssr/server/template/index.js

```diff
@@ -18,11 +18,17 @@ export const getTplParts = req => {
 export class HtmlBuilder {}
 
 HtmlBuilder.buildHead = ({headTplPart, helmetHead = {}}) => {
+  const headElement = headTplPart.substr(headTplPart.indexOf('<head'))
```

Comment:
Is this because we expect somebody would put attributes in the `<head>` element?

Reply:
Yes, for example in Infojobs the head element have some attrributte

Reply:
Thanks!